exercises/yacht/canonical-data.json

```diff
@@ -0,0 +1,260 @@
+                "dice":[2, 2, 2, 2, 2],
+                "category":"full house"
+            },
+            "expected":0
+        },
+        {  
+            "description":"Four of a Kind",
+            "property":"score",
+            "input":{  
+                "dice":[1, 2, 2, 2, 2],
+                "category":"four of a kind"
+            },
+            "expected":8
+        },
+        {  
+            "description":"Four of a Kind unsorted",
+            "property":"score",
+            "input":{  
+                "dice":[6, 6, 4, 6, 6],
+                "category":"four of a kind"
+            },
+            "expected":24
+        },
+        {  
+            "description":"Yacht can be scored as Four of a Kind",
+            "property":"score",
+            "input":{  
+                "dice":[3, 3, 3, 3, 3],
+                "category":"four of a kind"
+            },
+            "expected":12
+        },
+        {  
+            "description":"Not Four of a Kind",
+            "property":"score",
+            "input":{  
+                "dice":[3, 3, 3, 5, 5],
+                "category":"four of a kind"
+            },
+            "expected":0
+        },
+        {  
+            "description":"Little Straight",
+            "property":"score",
+            "input":{  
+                "dice":[1, 2, 3, 4, 5],
+                "category":"little straight"
+            },
+            "expected":30
+        },
+        {  
+            "description":"Little Straight unsorted",
+            "property":"score",
+            "input":{  
+                "dice":[3, 5, 4, 1, 2],
+                "category":"little straight"
+            },
+            "expected":30
+        },
+        {  
+            "description":"Little Straight as Big Straight",
+            "property":"score",
+            "input":{  
+                "dice":[1, 2, 3, 4, 5],
+                "category":"big straight"
+            },
+            "expected":0
+        },
+        {  
+            "description":"Not a little straight",
+            "property":"score",
+            "input":{  
+                "dice":[1, 2, 3, 4, 6],
+                "category":"little straight"
+            },
+            "expected":0
+        },
+        {  
+            "description":"Big Straight",
+            "property":"score",
+            "input":{  
+                "dice":[2, 3, 4, 5, 6],
+                "category":"big straight"
+            },
+            "expected":30
+        },
+        {  
+            "description":"Big Straight unsorted",
+            "property":"score",
+            "input":{  
+                "dice":[4, 6, 2, 5, 3],
+                "category":"big straight"
+            },
+            "expected":30
+        },
+        {  
+            "description":"Big Straight as little straight",
+            "property":"score",
+            "input":{  
+                "dice":[6, 5, 4, 3, 2],
+                "category":"big straight"
```

Comment:
little straight (given the test description is "Big Straight as little straight")